schema/components.json

```diff
@@ -371,6 +371,13 @@
                         "t3",
                         "e3",
                         "xdsl",
+                        "docsis",
+                        "gpon",
+                        "xg-pon",
+                        "xgs-pon",
+                        "ng-pon2",
+                        "epon",
+                        "10g-epon",
```

Comment:
Did not notice this before I brought in the pon ports.

Could you remove and then repush?  We still need DOCSIS added anyways